v23.1/full-text-search.md

```diff
@@ -0,0 +1,472 @@
+---
+title: Full-Text Search
+summary: Full-text searches using TSVECTOR and TSQUERY enable natural-language searches on documents with ranked results.
+toc: true
+docs_area: develop
+---
+
+A full-text search is used to perform natural-language searches on documents such as articles, websites, or other written formats.
+
+This page describes how to perform full-text searches using the provided [built-in functions](functions-and-operators.html#full-text-search-functions).
+
+{{site.data.alerts.callout_info}}
+Some PostgreSQL syntax and features are currently unsupported. For details, see [Unsupported features](#unsupported-features).
+{{site.data.alerts.end}}
+
+## How does full-text search work?
+
+In the PostgreSQL terminology, a *document* is a natural-language text, typically stored within a single database row or concatenated from multiple fields, that you [convert to a data type](#process-a-document) that is searchable using [specially formatted queries](#form-a-query).
```

Comment:
I wonder if two sentences here for the definition could remove the double "that" clauses. I am not an expert here, so hopefully my suggestion doesn't obscure meaning:

> In the PostgreSQL terminology, a *document* is a natural-language text [converted to a data type](#process-a-document) that is searchable using [specially formatted queries](#form-a-query). They/Documents/Converted data type(?) are typically stored within a single database row or concatenated from multiple fields.